dom.bs

```diff
@@ -4318,6 +4324,7 @@ To <dfn export>locate a namespace</dfn> for a <var>node</var> using
 
  <dt>{{DocumentType}}
  <dt>{{DocumentFragment}}
+ <dt>{{Attr}}
```

Comment:
Blink actually the equivalent of "Return the result of locating a namespace prefix for its element, if that is not null, and null otherwise"